src/survey/controllers/update_survey_controller.py

```diff
@@ -0,0 +1,30 @@
+from api.utils import failure_response
+from api.utils import modify_attribute
+from api.utils import success_response
+from rest_framework import status
+from survey.models import Survey
+
+
+# TODO(@chalo2000) Convert to celery task
+class UpdateSurveyController:
+    def __init__(self, id, data):
+        self._id = id
+        self._data = data
+
+    def process(self):
+        # Get the model
+        survey = Survey.objects.filter(id=self._id)
+        if survey is None:
+            return failure_response("Survey does not exist", status.HTTP_404_NOT_FOUND)
+        survey = survey[0]
+
+        # Extract attributes
```

Comment:
what about `did_meet` :0

Reply:
I think I had a reason for not including it but I'll put it in because this'll eventually be an admin route and we should have access to set every property to take care of the data on prod.